tensorflow_addons/seq2seq/README.md

```diff
@@ -77,19 +78,20 @@ import tensorflow_addons as tfa
 # Build RNN
 #   encoder_outputs: [max_time, batch_size, num_units]
 #   encoder_state: [batch_size, num_units]
-encoder = tf.keras.layers.LSTM(num_units)
+encoder = tf.keras.layers.LSTM(num_units, return_state=True)
 encoder_outputs, state_h, state_c = encoder(encoder_emb_inp)
 encoder_state = [state_h, state_c]
 
 # Sampler
-sampler = tfa.seq2seq.TrainingSampler()
+sampler = tfa.seq2seq.sampler.TrainingSampler()
```

Comment:
I think the __init__ file under seq2seq has been recently updated. So TrainingSampler is now available under tfa.seq2seq directly. Maybe the change has been reach the release yet.

I think this applies to all the symbols within this doc.

Reply:
Ah okay, good to know. I'll update the PR. I think the other changes are still valid, though (for example, the missing `return_states=True`), right?

Reply:
Correct. Thanks for catching them.